ts/src/test/test.ts

```diff
@@ -68,11 +67,6 @@ function exceptionMessage (exc) {
     return '[' + exc.constructor.name + '] ' + exc.message.slice (0, 500);
 }
 
-function addProxy (exchange, httpProxy) {
-    // add real proxy agent
-    exchange.agent = HttpsProxyAgent (httpProxy);
```

Comment:
question, now we don't need to do this but would work if we did right?

Reply:
.agent should work normally yeah, we haven't touched /changed its functionality

Reply:
we just added an extra unified way `proxyAgentCallback` to set it across 3 langs in their ways.

however, js specific way to set `.agent` customly as it was till date will remain ofc